examples/tutorial/gloo/texture_precision.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
```

Comment:
Can you add your name as author here, so that you're properly acknowledged and we know who to contact in case this example breaks :)